### ISSUE TITLE
Shapeworks 6.1 Testing: shapeworks module not found

### ISSUE THREAD

**Issue by patkins** [SCIInstitute/ShapeWorks#1280]
For #1273, it seems that the command `source ./conda_installs.sh` is not setting the path for the python shapeworks module properly. Is this something that should be manually done and included in the readme.txt file or was there a failure during install?
```
(shapeworks) atkins@localhost:/data/shapeworks/Examples/Python> python RunUseCase.py --use_case ellipsoid
Traceback (most recent call last):
  File "RunUseCase.py", line 19, in <module>
    import shapeworks as sw
  File "/home/atkins/miniconda3/envs/shapeworks/lib/python3.7/site-packages/shapeworks/__init__.py", line 5, in <module>
    from shapeworks_py import *
ModuleNotFoundError: No module named 'shapeworks_py'
```

**Comment by cchriste** [comment]
Thank you. We noticed this earlier and are working on it right now. In the meantime there is a workaround so you can continue testing:
`export PYTHONPATH=`/path/to/shapeworks/download/bin`:$PYTHONPATH`

**Comment by cchriste** [comment]
Having difficulty importing shapeworks_py on windows the same was that works on linux and osx, by just copying the .so to the python module installation directory in conda. It seems to need some other dll.

**Comment by cchriste** [comment]
> Having difficulty importing shapeworks_py on windows the same was that works on linux and osx, by just copying the .so to the python module installation directory in conda. It seems to need some other dll.

Figured this out using https://github.com/lucasg/Dependencies and identifying the local dlls that also needed to be copied. conda_install.bat now updated.

**Comment by cchriste** [comment]
Fixed and merged to master and new 6.1 branch.
